services/distribution/src/main/java/app/coronawarn/server/services/distribution/assembly/component/QrCodePosterTemplateStructureProvider.java

```diff
@@ -84,6 +84,10 @@ private QRCodePosterTemplateIOS buildIosProtoStructure(QrCodePosterTemplate temp
             .setOffsetY(textBoxConfig.getOffsetY().intValue()).setFontSize(textBoxConfig.getFontSize())
             .setHeight(textBoxConfig.getHeight()).setFontColor(textBoxConfig.getFontColor())
             .build())
+        .setAddressTextBox(QRCodeTextBoxIOS.newBuilder().setOffsetX(textBoxConfig.getOffsetX().intValue())
```

Comment:
@ioangut wouldn't that set the parameters on addressTextBox to the same values than the descriptionTextBox?